### ISSUE TITLE
[Bug] farmer response times in debug log are sometimes negative

### ISSUE THREAD

**Issue by brause** [Chia-Network/chia-blockchain#10900]
### What happened?

First off: My full node is running fine. Response times for peaks and challenges are fine and wins are within the expected timeframes.  So i am guessing there is a minor bug in gathering these numbers and i am not concerned in the very least.

These outliers do go down after a restart of the node but go to this after a day or two.

I looked a the farmer response times in the debug.log and got:
`lines   20284   average 8793.71 median  4.22665 min     -3.2203 max     783519.7746`
here are the 4 negative lines:
```
2022-03-22T01:19:10.915 full_node chia.full_node.full_node: INFO     Added unfinished_block d09453b8e7f551074d643fea44102cdb91678a03522aa54a635246c6f9658ee2, not farmed by us, SP: 11 farmer response time: -0.3926, Pool pk xch1sryqtpz36tzv9jmlyjrzyf05znyu0ytzcxax3tyta64dcpe8l65scujf7n, validation time: 0.1580 seconds, pre_validation time 0.0520, cost: 3873990225, percent full: 35.218%
2022-03-23T20:56:22.348 full_node chia.full_node.full_node: INFO     Added unfinished_block cc06158c49efbde85a4ffc80e62e05b7d6e958969720889a262d0591829c226a, not farmed by us, SP: 1 farmer response time: -0.0030, Pool pk xch1mehd8yuku8arsqau7cv0af6e68773efq6ad2mfqfc9k5d5a3u86q8cn9zy, validation time: 0.1440 seconds, pre_validation time 0.0580, cost: 3720559256, percent full: 33.823%
2022-03-24T03:08:26.658 full_node chia.full_node.full_node: INFO     Added unfinished_block eb7d004e4ef3577345ee40c7f45ba45376b17d674082c143ba79ab0d8c74ee65, not farmed by us, SP: 22 farmer response time: -3.0017, Pool pk xch1vyalvr8jnu8peuqhd9t2m35acpzy8kv8559rale8cfs7hgw5yndqvqvcx8, validation time: 6.6907 seconds, pre_validation time 0.1100, cost: 5306020346, percent full: 48.237%
2022-03-24T03:08:26.804 full_node chia.full_node.full_node: INFO     Added unfinished_block db19543caecae1e1a6474837cf5fb7fe01ab16c6c011f9d6395c3d20f4bc930d, not farmed by us, SP: 22 farmer response time: -3.2203, Pool pk xch1zckgl8vmqk5e28nuuq02g0jh7w3ptcza87fya0prr7lsa0n8942qynwv64, validation time: 0.1406 seconds, pre_validation time 1.7602, cost: 3622356134, percent full: 32.931%
```
and since we are already at it here are the lines around the "farmer response time: 783519.7746" line:
```
2022-03-26T09:04:56.238 full_node chia.full_node.full_node: INFO      Updated peak to height 1747594, weight 3692779168, hh 7643da4267181965ab4c7a58fcfad57aa2cf51ae6f2c3db015771f9cff51dc2f, forked at 1747593, rh: 9b35580ff9ebdbe68a7517b53a58d6be3f78cc2fb65ad565bb6c842829
4b1be1, total iters: 6854228739712, overflow: False, deficit: 0, difficulty: 2400, sub slot iters: 136314880, Generator size: No tx, Generator ref list size: No tx
2022-03-26T09:04:56.301 wallet chia.wallet.wallet_blockchain: INFO     Peak set to: 1747594 timestamp: 1648281858
2022-03-26T09:04:57.087 full_node chia.full_node.full_node: INFO       Finished sub slot, SP 64/64, 6243053402304605df649fc17c8925c2f4a423441feaaff41bd2ee5626629d5d, number of sub-slots: 2, RC hash: 740d4bb7ec80e74c6639293f08a90bb71bcdce24a4a289eacfc32f3af41444ca, Defici
t 16
2022-03-26T09:04:57.144 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming 6243053402... Found 0 proofs. Time: 0.04200 s. Total 457 plots
2022-03-26T09:05:01.350 full_node chia.full_node.full_node: INFO     Added unfinished_block 66a5601910daa683a64595ddd36b1eda4c284cfa39058c92f12b4ffeecfef191, not farmed by us, SP: 0 farmer response time: 783519.7746, Pool pk xch1vxs99eqerrsrk4axg92jm9tlcueqxj3ru5k6lxl8hg75awtfmjzskmj4ax, validation time: 0.1702 seconds, pre_validation time 0.0848, cost: 4250973044, percent full: 38.645%
2022-03-26T09:05:06.202 full_node chia.full_node.full_node: INFO       Finished signage point 1/64: CC: b8119984e6b8fdc37ccee5994c3a6bc20ccb777f3ce232d3430a0f0c847df16e RC: 414f4db24add15cf97e289d0cdcff70b805f9377c0c25d472578f5fcd5d521dd
2022-03-26T09:05:06.352 harvester chia.harvester.harvester: INFO     1 plots were eligible for farming 6243053402... Found 0 proofs. Time: 0.13700 s. Total 457 plots
2022-03-26T09:05:09.058 full_node full_node_server        : INFO     Connection closed: 188.19.57.1, node id: f8cc82e665c7010713283d9c0345be147225c1fdd8f75ee5c06e18b83e0ac6b0
2022-03-26T09:05:09.059 full_node chia.full_node.full_node: INFO     peer disconnected {'host': '188.19.57.1', 'port': 8444}
2022-03-26T09:05:10.713 full_node full_node_server        : INFO     Cannot connect to host 139.205.88.69:8444 ssl:<ssl.SSLContext object at 0x000001E7E5336F40> [Das Zeitlimit f<FC>r die Semaphore wurde erreicht]
2022-03-26T09:05:14.612 full_node chia.full_node.full_node: INFO       Finished signage point 2/64: CC: 11b34049bba16dcb65f4cc7eb3bf90d3f94750ba384c1d11cea5a356f347c5c4 RC: 8233c2a16eedfeaa8decc3b5118b782de904d5a4881f603f5d2de4f89406cea2
2022-03-26T09:05:14.953 harvester chia.harvester.harvester: INFO     2 plots were eligible for farming 6243053402... Found 0 proofs. Time: 0.32100 s. Total 457 plots
2022-03-26T09:05:23.527 full_node chia.full_node.full_node: INFO       Finished signage point 3/64: CC: cf4dd7de8bedaa860b02b291fc658bc1d2ebcbd13360e957f993880cd67ca378 RC: b66636924440508d8e8d0739aa2bfdef603c161e070e588d3f2370afb91e0b7a
2022-03-26T09:05:23.551 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming 6243053402... Found 0 proofs. Time: 0.01000 s. Total 457 plots
2022-03-26T09:05:33.335 full_node chia.full_node.full_node_store: INFO     Don't have rc hash 3dfdb33378741edb363226d4386ae3efb8eb074a4cf38320d039232573d34b27. caching signage point 4.
2022-03-26T09:05:36.887 full_node chia.full_node.full_node: INFO      Updated peak to height 1747595, weight 3692781568, hh a4cb0ba5c980556752f0e4420bf89497355b0ddbcf655b1fc99175314552371f, forked at 1747594, rh: 3dfdb33378741edb363226d4386ae3efb8eb074a4cf38320d039232573
```
Hope this helps.

### Version

1.3.1

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by brause** [comment]
version 1.5.x now: 
they are not negative anymore but still vary a lot from 1 sec to 400000 sec. Still seems way off but we are getting there somehow. last day:
![image](https://user-images.githubusercontent.com/6328682/187353927-1f741d00-5966-43f0-9d7d-00619be3eb0d.png)
and they are still building up if you do not restart the node.
on the left restart daily. then nothing because of sdcard failure on munin pi. right up again but no restarts and it keeps going up:
![image](https://user-images.githubusercontent.com/6328682/187354280-39c7d697-2b57-466a-a5d0-e2fe1b934130.png)

**Comment by brause** [comment]
connected to this issue, i guess:
https://github.com/Chia-Network/chia-blockchain/issues/11869#issuecomment-1154559962

**Comment by emlowe** [comment]
This is likely caused by the wrong time for SP: 0 which was fixed in #13655. Marking Closed - please add comments if the issues seems to continue
